### ISSUE TITLE
[Website]: Neither able to edit 'date' edit field in Command mode nor able to switch to Dictation mode

### ISSUE THREAD

**Issue by Shireen2000** [microsoft/AdaptiveCards#7508]
### Target Platforms

Other

### SDK Version

103.0.1255.0 

### Application Name

Adaptive Cards

### Problem Description

https://microsoft.visualstudio.com/OS/_queries/edit/39921438/?triage=true

Test Environment:
OS: 22H2 (OS Build 25120.1000)
Browser: Edge Dev
Browser Version: Version 103.0.1255.0 (Official build) dev (64-bit)
URL: [Adaptive Cards](https://adaptivecards.io/)
Tool: Voice Access

Repro Steps:
Launch the above URL.
Command: "Voice access wake up"
Command: "Click Schema Explorer" for the top header.
Command: 'Click Input.Date' from under 'Inputs' label.
Navigate till the first card present under this page.
Command: 'Click Enter a Date.
Command: 'Dictation Mode'.
Command: 'Command Mode'.
Command: 'Type 10'.
Observe the issue.
Actual Result:
Users are not able to edit 'Enter a Date ' edit field using 'Type' command.
Also, we are not able to switch to Dictation mode when our focus is on 'Enter a Date' edit field.

Expected Result:
Voice Access users should be able to edit 'Enter a Date' edit field using 'Type 10'. Users should also be able to switch to Dictation mode.

Note: This issue is repro throughout this page.

Observation: 'IsKeyboardFocusable' property is set as 'True' for 'Enter a Date' edit field.
Keyboard user is able to edit in 'date' edit field.

User Experience:

People with limited mobility (cerebral palsy, quadriplegia) who use Voice Access to use OneNote may not be able edit or type they will find difficulty in accessing 'Canvas area' and 'Canvas Title' edit field. Voice access users will get impact and will not be able to interact with controls with pre-defined Voice access commands.
 
MAS Reference:
[No Disruption of Accessibility Features - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.03.01)
 
Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

To learn more about Voice Access and how an application can support its different modes, Refer: [VoiceAccess Testing - OSGWiki](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.osgwiki.com%2fwiki%2fVoiceAccess_Testing&data=05%7c01%7cv-shiwasi%40microsoft.com%7c00297be07a15467b634408da325a247c%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637877659989570305%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=sQj%2Bb4UzcB2jUBsX3r%2f4%2BjkjzUrXncXKGhGCIuu8vc4%3D&reserved=0)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
As you can see in the following code snippet, we have rendered an HTML Input element with the type `date`.

```HTML
<input type="date" placeholder="Enter a date" aria-label="Enter a date" class="ac-input ac-dateInput" id="__ac-e5f40692-f710-40a6-ac94-f962c80bf320" style="width: 100%; min-width: 0px;" tabindex="4">
```

We don't have control over the inner elements of the date input. You can see this issue also occurs within [documentation examples](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date#try_it) for the control.

**Comment by anna-dingler** [comment]
Closing the issue since we determined offline that this is an issue with Voice Access.
